ui/README.md

```diff
@@ -81,7 +81,7 @@ The main dashboard provides comprehensive request monitoring:
 
 Manage all your AI providers from a unified interface:
 
-- **Supported Providers**: OpenAI, Azure, Anthropic, AWS Bedrock, Cohere, Google Vertex AI, Mistral, Ollama, Parasail, Elevenlabs, SGLang, Cerebras, Groq, Gemini, OpenRouter
+- **Supported Providers**: OpenAI, Azure, Anthropic, AWS Bedrock, Cohere, Google Vertex AI, Mistral, Ollama, Parasail, Elevenlabs, SGLang, Cerebras, Groq, Gemini, OpenRouter, HuggingFace
```

Comment:
@qwerty-dvorak lets update this entire readme (its very outdated) and instead of manually maintains list of features and provider name - lets redirect everything to the docs (like we do in the main readme).